### ISSUE TITLE
Linux support (via WINE)

### ISSUE THREAD

**Issue by RustoMCSpit** [DamRsn/NeuralNote#76]
(this is just a discussion issue)

**Comment by DamRsn** [comment]
I can leave this open if you want, but note that I won't support NeuralNote on Linux via wine.

**Comment by RustoMCSpit** [comment]
> I can leave this open if you want, but note that I won't support NeuralNote on Linux via wine.

yeah just leave it open for others to talk on / prevent duplicates

**Comment by ilikecatgirls** [comment]
I couldn't get the standalone executable to run via a prefix I use for music production, but on the same prefix, it ran fine on FL Studio as a VST.
